src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -124,6 +125,14 @@ const routes: Routes = [
           resource: 'api.cluster.user@1.0'
         }
       },
+      {
+        path: 'cluster/user/create',
+        component: CrudFormComponent,
+        data: {
+          breadcrumbs: 'Cluster/Users',
+          resource: 'api.cluster.user@1.0'
+        }
+      },
```

Comment:
This is ok for the time being, but what about trying [adding routes dynamically](https://levelup.gitconnected.com/angular-dynamic-routing-299c04ca75b1)? The backend information is available when the UI application bootstraps, so it should be easy to add new routes from a back-end endpoint without having to touch this file.

Reply:
https://tracker.ceph.com/issues/58428